src/mousetracker.js

```diff
@@ -2823,10 +2832,7 @@
             // If child element relinquishes capture to a parent we may get here
             //   from a pointerleave event while a pointerup event will never be received.
             //   In that case, we'll clean up the contact count
-            if ( (pointsList.type === 'mouse' || pointsList.type === 'pen') &&
-                                                        pointsList.contacts > 0 ) {
-                pointsList.removeContact();
-            }
+            pointsList.removeContact();
```

Comment:
Can you explain why this fixes the issue?

Reply:
removeContacts is called twice, and the the same if statement was needed in both places. I moved the if statement inside removeContacts